website/src/_data/authors.yml

```diff
@@ -22,6 +22,9 @@ aljoscha:
 altay:
     name: Ahmet Altay
     email: altay@apache.org
+angoenka:
```

Comment:
I suppose this is the github username and not the apache username.

Reply:
I think it is just an arbitrary identifier used by the website generator.